README.md

```diff
@@ -1,9 +1,27 @@
 # TMAP8
 Tritium Migration Analysis Program (TMAP8)
 
-TMAP8 is a MOOSE application for solving 0-1D scalar diffusion-reactive transport. It contains unique custom syntax for 0D or 1D geometric/algebraic entities that make it more usable for 0-1D transport than standard MOOSE syntax.
-
-The TMAP8 documentation, which includes the code reference, a getting started guide, and details about verification, validation, and examples case, etc. can be found [here](https://mooseframework.inl.gov/tmap8).
+TMAP8 is an open-source, MOOSE-based application developed to provide state-of-the-art tritium transport and fuel cycle modeling capabilities.
+TMAP8 aims to expand the capabilities of previous versions (i.e., TMAP4 and TMAP7) by leveraging modern computational techniques, ensuring high software quality assurance standards (key to building trust), and enabling multispecies, multiscale, and multiphysics simulations for integrated tritium transport modeling in complex geometries.
```

Comment:
```suggestion
TMAP8 aims to expand the capabilities of previous versions (i.e., TMAP4 and TMAP7) by leveraging modern computational techniques; ensuring high software quality assurance standards (key to building trust); and enabling multispecies, multiscale, and multiphysics simulations for integrated tritium transport modeling in complex geometries.
```

Since you have a list in your list, swap for semicolons.